cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -80,10 +81,31 @@ def __init__(self, message: str) -> None:
 
 
 def get_id_from_url(api_url: str) -> str:
-    """Get the PK from an API url"""
+    """Get the PK from an API url
+
+    :param api_url: api url with a pk
+    :return: list of opinion cluster objects
```

Comment:
This doesn't seem right, is it?

Reply:
It's already fixed and merged

Reply:
Thank you Kevin! I appreciate your continued enhancements to this script. It seems like it's becoming a popular one!